search-api/src/main/java/edu/unc/lib/boxc/search/api/requests/SearchState.java

```diff
@@ -290,15 +290,18 @@ public RangePair(String leftHand, String rightHand) {
         }
 
         public RangePair(RangePair rangePair) {
-            this.leftHand = rangePair.getLeftHand();
-            this.rightHand = rangePair.getRightHand();
+            String leftHand = rangePair.getLeftHand();
```

Comment:
Seems like we don't need to set to local variables first before setting the instance variables, so might be able to revert changes to this method.